### PR TITLE
Updated json for v3 purchase order API's

### PR DESCRIPTION
**Pull request by harishrconcur**

### CODE REVIEW

File: src/api-reference/invoice/v3.purchase-order-receipt.markdown [hunk 1]
```diff
@@ -39,7 +39,7 @@ Name|Type|Format|Description
 
 ```json
 {
-  "Deleted": false,
+  "Deleted": "false",
```
Comment (MarcBell-SAP) [inline src/api-reference/invoice/v3.purchase-order-receipt.markdown:42]: Is this a String that's using boolean logic?
If not, the boolean value should not be within quotation marks.

File: src/api-reference/invoice/v3.purchase-order.markdown [hunk 2]
```diff
@@ -78,42 +78,50 @@ Create or update a Purchase Order. Batch processing is not available using the P
...
+    "Address3": "add3",    
     "City": "cityship",
     "CountryCode": "US",
     "ExternalID": "Shiptoapi",
     "Name": "shiptoapi",
     "PostalCode": "55426",
-    "State": "CA",
     "StateProvince": "MN"
   },
+  "Status": "Pending",
```
Comment (MarcBell-SAP) [inline src/api-reference/invoice/v3.purchase-order.markdown]: "Pending" is in an invalid value for the "Status" property. The purchase order schema states:

> Supported values: Closed, Transmitted

File: src/api-reference/invoice/v3.purchase-order.markdown [hunk 3]
```diff
@@ -78,42 +78,50 @@ Create or update a Purchase Order. Batch processing is not available using the P
   "CurrencyCode": "USD",
   "OrderDate":"2011-08-12T20:17:46.384Z",
   "ID": "API101",
-  "IsTest": "N",
+  "IsTest": "false",
```
Comment (MarcBell-SAP) [inline src/api-reference/invoice/v3.purchase-order.markdown]: Is this a String that's using boolean logic?
If not, the boolean value should not be within quotation marks.

**Review by MarcBell-SAP — CHANGES_REQUESTED** [review]

File: src/api-reference/invoice/v3.purchase-order-receipt.markdown (hunk 1 above)
Comment (harishrconcur) [reply src/api-reference/invoice/v3.purchase-order-receipt.markdown:42]: Fixed in 51a69ac

File: src/api-reference/invoice/v3.purchase-order.markdown (hunk 2 above)
Comment (harishrconcur) [reply src/api-reference/invoice/v3.purchase-order.markdown]: Fixed in 51a69ac

File: src/api-reference/invoice/v3.purchase-order.markdown (hunk 3 above)
Comment (harishrconcur) [reply src/api-reference/invoice/v3.purchase-order.markdown]: Fixed in 51a69ac

**Review by MarcBell-SAP — APPROVED** [review]

**Pull request merged**